packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -6,20 +6,46 @@ import { noSuchObject } from 'xo-common/api-errors'
 import config from '../_config'
 import randomId from '../_randomId'
 import xo from '../_xoConnection'
+import { getDefaultName, getDefaultSchedule } from '../_defaultValues'
 
-const DEFAULT_SCHEDULE = {
-  name: 'scheduleTest',
-  cron: '0 * * * * *',
+// it only support one schedule
```

Comment:
Is it hard to fix this?

Reply:
Yes because we need to match the old schedule to the new one which has a new id.